### ISSUE TITLE
[Suggestion] AutoCrafting

### ISSUE THREAD

**Issue by Xaikii** [KidsDontPlay/PlayerStorage#7]
I think nobody can complain about this mod, unbelievable handy(maybe for some packs unbalanced) but for me perfect, meanwhile I use this mod I noticed 1 things... I am kinda missing the AutoCrafting Part because I have replaced AE2 with this Mod in my pack, and some players can't / won't play without autocrafting.

Meanwhile the mod does remember about some Cyborg/Android who is able to have a whole Storage with him, my Idea was to have an extra item slot in the GUI for a special Item, well basically a Chip with a duration. Each Crafting will lower the duration by 1. 

Optimal the Damage Value of the Chips would be configurable ^^

About the Chip Types, I thought about were:
1 Crafting all 5 Ticks
1 Crafting all 2 Ticks
1 Crafting every Tick
3 Crafting every Tick
5 Crafting every Tick
(I think everything beyond would cause lags)

Also while I am at it if this Suggestion will be accepted would it be possible to have a support for ExtendedCrafting? So that 3x3, 5x5, 7x7 & 9x9 Recipes can be automated? 
Basically every Size would have it's own Chips, all would have the speed meantioned above but in different tiers(3x3 Tier 1, 5x5 Tier 2, etc,.

Greetings

**Comment by KidsDontPlay** [comment]
I thought about adding auto crafting. but I wasn't able to implement it. It's quite complicated.
the idea with the chip sounds good. you are not able to craft items (even manually) without a crafting chip.

**Comment by Xaikii** [comment]
ah ok :/
